### PR TITLE
Drop PHP rankings in favor of RoR ones.

### PR DESCRIPTION
**Pull request by AlbertoPdRF**
After having checked that the code from #4394 works on production (see [here](https://www.worldcubeassociation.org/new-results/rankings)) it's time to get rid of the old rankings page!

The changes on `chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb` come from @jonatanklosko (see https://github.com/thewca/worldcubeassociation.org/pull/4394#discussion_r312718940,) and the changes from this PR are currently deployed on staging for if someone wants to check them out :slightly_smiling_face:

### CODE REVIEW

File: WcaOnRails/app/views/persons/_personal_records.html.erb [hunk 1]
```diff
@@ -28,12 +28,12 @@
             <%= rank_td rank_single, "continent" %>
             <%= rank_td rank_single, "world" %>
             <td class="single">
-              <%= link_to_ranking rank_single.event_id, :single do %>
+              <%= link_to rankings_path(rank_single.event_id, "single"), class: "plain" do %>
```
Comment (jonatanklosko) [inline WcaOnRails/app/views/persons/_personal_records.html.erb]: Great you got path helpers to work =)

Comment (AlbertoPdRF) [reply WcaOnRails/app/views/persons/_personal_records.html.erb]: They work as long as there's an `as: :something` on the route, I couldn't make them work without it :confused:

**Comment by AlbertoPdRF** [comment]
While playing around on staging I noticed that old links to a region with spaces on it (e.g. South America) are also of the form `region=_South%2BAmerica`, so I've just added a commit to handle that properly. Everything else seems to work as expected!

File: chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb [hunk 2]
```diff
@@ -155,6 +155,45 @@ server {
     return 302 /media/;
   }
 
+  # Redirect e.php to results/events.php
```
Comment (jfly) [inline chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb]: nit: This is actually "Redirect /results/e.php to ..."

File: chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb [hunk 3]
```diff
@@ -155,6 +155,45 @@ server {
...
+  location /results/e.php {
+    return 302 /results/events.php$is_args$args;
+  }
+  # Redirect events.php to /results/rankings
```
Comment (jfly) [inline chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb]: nit: This is actually "Redirect /results/events.php to ..."

File: chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb (hunk 2 above)
Comment (AlbertoPdRF) [reply chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb]: Right, updated!

File: chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb (hunk 3 above)
Comment (AlbertoPdRF) [reply chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb]: Right, updated!

**Comment by AlbertoPdRF** [comment]
Thanks for the reviews! I'll merge this one when tests pass :smile:

**Pull request merged**